### DISCUSSION TITLE
[SOLVED] Logged in with wrong account on Visual Studio 2022

### DISCUSSION THREAD

**Discussion by Agumons** [community/community#13982] [Copilot Conversations] [answered]
I logged in with my second github account, which is not authorized. I would like to use it with this one but it seems like I can't.
I tried reinstalling the extension again but it's not working ( i keep getting the message "You don't have access to GitHub Copilot.").
How can I solve this issue?
Thanks in advance!

Edit: After some trial and error I was able to solve the problem!

1- with the extension installed I manually deleted the folder "C:\Users\[username]\AppData\Local\github-copilot" (which now for some reason is no longer there)
2- I opened Visual Studio that finally presented me the popup that gave me the possibility to do the verification from my enabled account
3- I uninstalled and reinstalled the extension

Huge thanks to this Stack Overflow user!
https://stackoverflow.com/questions/19115829/how-to-permanently-remove-extension-from-visual-studio-2012

**Comment by kzu** [comment]
The device auth page should have opened (typically as an extra tab) in your (likely already opened) default browser window. If you're like me, perhaps it got lost among 42 other tabs :). You can directly navigate to https://github.com/login/device and paste the code there though 👍

**Reply by ChaseRichardsDev** [reply]
I have OPs issue as well. The problem is not getting the code. We have already gotten the code before and linked it to the wrong account. The extension is stuck in a state where it will not unlink the wrong account.

**Reply by Agumons** [reply]
Thanks for the reply! 
Unfortunately I don't have the code and the problem is that Visual Studio seems to be linked to the account I previously logged in with (the wrong one). After reinstalling the extension I didn't get the auth page for this other account. 
I also tried reinstalling the IDE from scratch but could not get Copilot to work.

**Reply by ChaseRichardsDev** [reply]
Copilot seems to be holding onto that first login somewhere and using it. There is nowhere that I can find to have Copilot let you select another login. I've done everything you have tried as well.

**Reply by kzu** [reply]
Ok, looks like it should indeed clear up some previous state before retrying the sign-in....

**Comment by Agumons** [comment, marked as answer]
See the edit!

**Reply by ChaseRichardsDev** [reply]
Thank you!

**Reply by FuSioN79m** [reply]
Big thanks to you! I had the same issue! Solved!

**Comment by tzanislav** [comment]
I don't know if it's exactly the same issue but I resolved mine by finding the tinyest button and going into the Add GitHub account. 
![image](https://github.com/community/community/assets/32868846/e193921a-0dc1-497e-976a-7801300efe8b)
